### PR TITLE
Exposing equivalence check to CLI

### PR DESCRIPTION
**Pull request by dwRchyngqxs**

### CODE REVIEW

File: src/Verismith.hs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 -- |
 -- Module      : Verismith
 -- Description : Verismith
--- Copyright   : (c) 2018-2019, Yann Herklotz
+-- Copyright   : (c) 2018-2022, Yann Herklotz
```
Comment (ymherklotz) [inline src/Verismith.hs:6]: You can add your name there too

File: src/Verismith.hs [hunk 2]
```diff
@@ -283,6 +283,25 @@ handleOpts (DistanceOpt v1 v2) = do
...
+    cp v1 (o </> fn1)
+    cp v2 (o </> fn2)
+  res <- shelly . runResultT $ pop o $ do
+      runEquiv checker (toFP datadir) (mkid fn1) (mkid fn2) src
+  case res of
+    Pass _ -> putStrLn "Equivalence check passed"
+    Fail (EquivFail _) -> putStrLn "Equivalence check failed"
+    Fail TimeoutError -> putStrLn "Equivalence check timed out"
+    Fail _ -> putStrLn "Equivalence check error"
+  return ()
```
Comment (ymherklotz) [inline src/Verismith.hs:300]: I guess you don't need this line, `putStrLn` returns type `IO ()` already.

**Review by ymherklotz — COMMENTED** [review]
Yeah all looks good, let me know if you want me to merge or if you wanted to edit it slightly first with the tiny comments I gave.

File: src/Verismith.hs (hunk 1 above)
Comment (dwRchyngqxs) [reply src/Verismith.hs:6]: I might do it in the future, not for changes this small though.

File: src/Verismith.hs (hunk 2 above)
Comment (dwRchyngqxs) [reply src/Verismith.hs]: Removed, same with the useless binding `_ <- shelly [...]` at line 289.

**Pull request merged**